ui/src/app/utils/access-utils.tsx

```diff
@@ -29,6 +29,21 @@ import {
   displayDateWithoutHours,
   MILLIS_PER_DAY,
 } from './dates';
+import {
+  hasExpired,
+  isExpiringNotBypassed,
+} from 'app/pages/access/access-renewal';
+
+export enum AccessModulesStatus {
+  // Complete represents modules that cannot be expired (eg ERA COMMON, LOGIN GOV etc) and are not bypassed/incomplete
+  COMPLETE = 'Complete',
```

Comment:
I'm not sure this works, because we already use the word "complete" for all modules, regardless of whether they expire